api-module-library/sharepoint/jest-env

```diff
@@ -0,0 +1,4 @@
+SHAREPOINT_SCOPE=sharepoint_scope_test
```

Comment:
What's this file?

I think I want to move towards .env for actual test runs (loaded with dotenv) and .env.example for collaboration purposes